ui/dist/index.html

```diff
@@ -8,8 +8,34 @@
     />
     <link rel="stylesheet" type="text/css" href="/index.css" />
     <link rel="icon" type="image/png" href="/img/logo-only.svg" />
+
+    <!-- Google Tag Manager -->
+    <script>
+      (function (w, d, s, l, i) {
+        w[l] = w[l] || [];
+        w[l].push({ 'gtm.start': new Date().getTime(), event: 'gtm.js' });
+        var f = d.getElementsByTagName(s)[0],
+          j = d.createElement(s),
+          dl = l != 'dataLayer' ? '&l=' + l : '';
+        j.async = true;
+        j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
+        f.parentNode.insertBefore(j, f);
+      })(window, document, 'script', 'dataLayer', 'GTM-5JMBB8K');
```

Comment:
Should we add an environment variable for the ID value, like GOOGLE_TAG_MANAGER_ID or so?

Reply:
I don't think we have a way of injecting environment variables. I can try to come up with one.

Reply:
OK I added code to run html files through `html/template` so we can use an environment variable for the id.